db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create(email: "admin@example.com", password: "password")
```

Comment:
Seeds should be [idempotent](https://www.google.com/search?q=idempotent) preferably, and reserved for DB records needed for the application functionality. Debug/Development users like this should rather be defined in a rake task.